pages/custom-query-modules/cpp/cpp-api.md

```diff
@@ -2354,6 +2354,54 @@ Additionally, operator<< is overloaded for Type enum, and usage of this operator
 std::ostream &operator<<(std::ostream &os, const mgp::Type &type)
 ```
 
+## Database internals API
+
+This section covers the C++ API methods for interfacing with the database other
+than with the graph itself.
+
+### Text search
+
+#### SearchTextIndex
+
+Search the named text index for the given query and get a list of the nodes
+whose text-indexed properties match the given query.
```

Comment:
Query the specified text index and retrieve a list of nodes with properties matching the query in the text index.